api/config/default.json

```diff
@@ -1,7 +1,7 @@
 {
   "server_host": "0.0.0.0:8000",
   "open_id": {
-    "host_url": "http://localhost:3000",
+    "host_url": "https://localhost:3000",
     "providers": {
       "google": {
         "issuer_url": "https://accounts.google.com"
```

Comment:
this is really minor and doesnt need to be addressed in this pr but should we move `host_url` to the root of the config?

Reply:
I was actually thinking the same thing. Right now we're just using it for OpenID stuff so I think it's fine where it is but if we start using it for anything else we should def move it out. I'm fine either way tho